TorchSharp/TorchSharp.csproj

```diff
@@ -48,5 +52,13 @@
     <Compile Update="TypeGeneration.cs">
       <DependentUpon>TypeGeneration.tt</DependentUpon>
     </Compile>
+    <Compile Update="FileOperations.cs">
+      <DependentUpon>FileOperations.tt</DependentUpon>
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+    </Compile>
+  </ItemGroup>
+  <ItemGroup>
+    <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
```

Comment:
I think we can remove this.

Reply:
The Service tag?

Reply:
I don't know why it keeps coming back. I've removed it before.